pkg/apis/config/v1alpha1/multi_cluster_service.go

```diff
@@ -0,0 +1,42 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MultiClusterService is the type used to represent the mesh configuration.
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type MultiClusterService struct {
+	// Object's type metadata.
+	metav1.TypeMeta `json:",inline" yaml:",inline"`
+
+	// Object's metadata.
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
+
+	// Spec is the MultiClusterService specification.
+	// +optional
+	Spec MultiClusterServiceSpec `json:"spec,omitempty" yaml:"spec,omitempty"`
+}
+
+type MultiClusterServiceSpec struct {
```

Comment:
Could we add comments to this file like this : https://github.com/openservicemesh/osm/blob/b223d9ed3fb50ce6fcc09842d1bba5fba6238482/pkg/apis/config/v1alpha1/mesh_config.go#L40 

This would help with auto generating the docs later